File: sqlUPDATE.php
<?php
$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";

$conn = new mysqli($servername, $username, $password, $dbname);

if($conn->connect_error){
    die("connection failed:".$conn->connect_error);
}

$sql = "UPDATE students SET name='cole' WHERE name='logan'";
$result =$conn->query($sql);

if ($conn->query($sql) === TRUE) {
    echo "Record updated.";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>

File: sqlUPDATE.py
import MySQLdb
#Open database connection
db =  MySQLdb.connect (host="localhost",user="User",passwd="password",db="school")
#prep a cursor object using cursor() method; cursor is like a navigation method
db.autocommit(True)
cursor = db.cursor(MySQLdb.cursors.DictCursor)

name='will'
age=12
gradeLevel=10

sql = "UPDATE students SET age=17 WHERE name='logan'"
cursor.execute(sql)

cursor.close()
db.close()
